webextension/privileged/ui/CloudDownloadsView.jsm

```diff
@@ -328,10 +319,24 @@ var CloudDownloadsView = {
     const fragment = document.createDocumentFragment();
     panelCloudNotification = document.createElement("vbox");
     panelCloudNotification.setAttribute("id", "panelCloudNotification");
-    panelCloudNotification.unsafeSetInnerHTML(this.notificationHTML);
     fragment.appendChild(panelCloudNotification);
     panelDownload.prepend(fragment);
 
+    // panelCloudNotification.innerHTML = this.notificationHTML;
```

Comment:
@gijsk Using innerHTML fails to render notification, with  'Flattening unsafe node (descendants are preserved)' message in logs, am I missing something here?